app/components/atomic-design/molecules/Navigation/Navigation.tsx

```diff
@@ -61,7 +57,7 @@ function NavigationMenu({
               {item.label}
             </Link>
 
-            {item?.children?.items?.length > 0 && (
+            {item?.children?.items && (
```

Comment:
@Danishkhurshid If any empty array was provided, wouldn't this still get triggered (which we don't want to happen)?